scripting/lua/LuaScriptModule.cpp

```diff
@@ -17,7 +17,7 @@
 #define strcpy_s(a, b, c) strncpy(a, c, b)
 #endif
 
-const char *g_cszAiName = "Lua interpreter";
+static const char *g_cszAiName = "Lua interpreter";
```

Comment:
Frankly, the only useful change I noticed so far

Reply:
This change is not from modernization, I done it by hand, because Lua do not builds without it)